src/test/functional/steps/phone-numbers.ts

```diff
@@ -79,20 +79,78 @@ Then('the phone number entry in last position has description at index {int} num
   expect(lastExplanationCyText).equal(explanationCy);
 });
 
-Then('a green update message is displayed in the phone numbers tab', async() => {
-  const elementExist = await I.checkElement('#phoneNumbersTab .govuk-panel--confirmation');
-  expect(elementExist).equal(true);
+Then('a green message is displayed for updated entries {string}', async (message: string) => {
+  const selector = '#phoneNumbersContent > div > h1';
+  expect(await I.checkElement(selector)).equal(true);
+
+  const messageUpdate = await I.getElement('#phoneNumbersContent > div > h1');
+  expect(await I.getElementText(messageUpdate)).equal(message);
 });
 
-When('I enter an incomplete phone number entry', async () => {
+When('I left description entry blank in phone number tab and enter phone number {string}', async (number: string) => {
   const numFieldsets = await I.countElement('#phoneNumbersTab fieldset');
   const entryFormIdx = numFieldsets - 2;
 
   // Ensure last entry is for new number and the description is unselected
-  const selectedDescription = await I.getLastElementValue('select[name$="[type_id]"');
+  const selectedDescription = await I.getLastElementValue('#phoneNumbersTab select[name$="[type_id]"');
   expect(selectedDescription).equal('');
 
-  await I.setElementValueAtIndex('input[name$="[number]"', entryFormIdx, '0987 666 5040');
+  await I.setElementValueAtIndex('#phoneNumbersTab input[name$="[number]"', entryFormIdx, number);
+});
+
+When('I left the phone number entry blank and select description at index {int}', async (descriptinIndex: number) => {
+  const numFieldsets = await I.countElement('#phoneNumbersTab fieldset');
+  const entryFormIdx = numFieldsets - 2;
+
+  const selectSelector = '#phoneNumbersTab select[name$="[type_id]"]';
+  const numbersSelector = '#phoneNumbersTab input[name$="[number]"]';
+
+  await I.setElementValueAtIndex(selectSelector, entryFormIdx, descriptinIndex, 'select');
+  await I.setElementValueAtIndex(numbersSelector, entryFormIdx, '', "input");
```

Comment:
The linter specifies that rather than use "" you need to use ''
```suggestion
  await I.setElementValueAtIndex(numbersSelector, entryFormIdx, '', 'input');
```